app/src/organisms/NetworkSettings/SetWifiSsid.tsx

```diff
@@ -68,15 +48,19 @@ export function SetWifiSsid({
         <InputField
           aria-label="wifi_ssid"
           value={inputSsid}
+          id="wifiSsid"
           onChange={e => setInputSsid(e.target.value)}
           type="text"
-          css={SSID_INPUT_FIELD_STYLE}
           error={errorMessage}
+          onBlur={e => e.target.focus()}
+          autoFocus={true}
```

Comment:
nit: no biggie but i think, since its a boolean, you don't need to specify if its true or not

```suggestion
          autoFocus
```

Reply:
I wasn't aware of this syntax in JSX— thanks